pages/workspaces/[workspaceId]/contributor-insights/[listId]/edit.tsx

```diff
@@ -90,10 +90,9 @@ export default function ContributorInsightEditPage({
   featureFlags,
   username,
 }: ContributorInsightEditPageProps) {
-  const showOscr = featureFlags && featureFlags["oscr-rating"];
   const {
     data: { data: contributors },
-  } = useContributorsList({ workspaceId, listId: list?.id, showOscr, username });
+  } = useContributorsList({ workspaceId, listId: list?.id, showOscr: true, username });
```

Comment:
This is always true in this case as the user is logged in since they're editing their own contributor insight.